client/app/bundles/shared/components/Footer.scss

```diff
@@ -0,0 +1,103 @@
+@import 'base';
+
+.footer {
+  width: 100%;
+  height: 450px;
+  background: rgba(0, 0, 0, 0.5);
+  font-family: $font-family;
+
+  .row {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+    margin-left: 5%;
+    margin-right: 5%;
+
+    @media screen and (max-width: $small) {
+      font-size: 14px;
+    }
+
+    .if_me {
+      text-align: left;
+
+      @media screen and (max-width: $small) {
+        margin-left: -30px;
+      }
+    }
+    
+    .connect {
+      text-align: center;
+    }
+    
+    .resources {
+      text-align: right;
+      margin-right: 30px;
+
+      @media screen and (max-width: $small) {
+        margin-right: 50px;
+      }
+    }
+
+    .love_foss {
+      text-align: right;
+      color: $background-color;
+      margin-right: 30px;
+      font-family: $font-family;
+      font-size: 20px;
+
+      p {
+        font-size: 18px;
+      }
+
+      @media screen and (max-width: $small) {
+        font-size: 14px;
+        margin-right: 20px;
+        margin-top: -15px;
+
+        p {
+          font-size: 12px;
+          margin-top: -20px;
+        }
+      }
+
+      @media screen and (max-width: $medium) {
+        font-size: 16px;
+
+        p {
+          font-size: 14px;
+        }
+      }
+    }
+    
+    .table_cell {
+      width: 26%;
+
+      ul {
+        margin-top: 50px;
+
+        .footer_header {
+          color: $header-text-color;
+          text-transform: uppercase;
+          margin-bottom: 10px;
+        }
+      }
+      
+      ul > li {
+        list-style: none;
+        line-height: 24px
+      }
+      
+      ul li a {
+        text-decoration: none;
+        color: $background-color;
+        letter-spacing: 1px;
```

Comment:
Let's take out the letter-spacing here! I just checked the Figma CSS, and the 0.05em that was recommended was likely just for the column label.